resources/fcontext.rb

```diff
@@ -5,3 +5,4 @@
 default_action :addormodify
 attribute :file_spec, :kind_of => String, :name_attribute => true
 attribute :secontext, :kind_of => String
+attribute :filetype, :kind_of => String, :equal_to => ['a', 'f', 'd', 'c', 'b', 's', 'l', 'p']
```

Comment:
We have a "filetype" attribute. "filetype: nothing" doesn't make any sense. Me and you know the default actually means "everything" because of implementation, but it's not obvious from the resource.
I don't like omitting it for any of these reasons - I appreciate you're trying to avoid breakage but if this passes in our kitchen framework I'm not worried.